src/Button/Button2.stories.tsx

```diff
@@ -151,9 +157,16 @@ export const blockButton = ({...args}: ButtonProps) => {
   )
 }
 
-export const disabledButton = ({...args}: ButtonProps) => {
+export const DisabledButton = ({...args}: ButtonProps) => {
```

Comment:
Nice! 

Can we also sneak in a quick rename to `Button.stories.tsx`? 😄

Reply:
did that in the other PR I have open!